src/coarse/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# Copyright (C) 2024 Intel Corporation
+# Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+include(${UMF_CMAKE_SOURCE_DIR}/cmake/helpers.cmake)
+
+set(COARSE_SOURCES coarse.c ../ravl/ravl.c)
+
+if(COARSE_BUILD_SHARED_LIBRARY)
```

Comment:
What does it mean? Can we build it as a shared library?

Reply:
It is always a static inner/helper library.

Reply:
Than it is fine :)